repos/spack_repo/builtin/packages/casacore/package.py

```diff
@@ -100,6 +103,11 @@ def cmake_args(self):
         args.append(self.define_from_variant("USE_MPI", "mpi"))
         args.append("-DPORTABLE=ON")  # let Spack determine arch build flags
 
+        # Use casacore-measures data directory if available
+        if spec.satisfies("^casacore-measures"):
+            measures_spec = spec["casacore-measures"]
+            args.append(self.define("DATA_DIR", measures_spec.prefix.share.data))
```

Comment:
Setting `DATA_DIR` during the build, whether it's for testing or not, kind of requires `casacore-measures` to be available during runtime, or applications might fail unexpectedly. Yes, users can override the measures location, but I think that by default, they should not have to if we've built casacore with a valid `DATA_DIR` value.